scale_out/test/MeshMultiCgraRTL_test.py

```diff
@@ -80,8 +80,8 @@ def test_homo_2x2_2x2(cmdline_opts):
   cgra_rows = 2
   cgra_columns = 2
   num_terminals = cgra_rows * cgra_columns
-  x_tiles = 2
-  y_tiles = 2
+  x_tiles = 4
+  y_tiles = 4
```

Comment:
We target 4 Cgra and each Cgra contains 16 tiles, that's why you change this, right?

Can we just copy this func and rename that as `test_homo_2x2_4x4`, and change these two params there?

Reply:
Thanks Ron, I saw you created new test, then plz change these `4` back to `2`.